### DISCUSSION TITLE
The log and CMD window show that the server is running, but the Github server cannot be attached.

### DISCUSSION THREAD

**Discussion by enchic** [modelcontextprotocol/.github#53] [Q&A]
### Pre-submission Checklist

- [X] I have checked that this question would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions and documentation for answers

### Question Category

- [ ] Protocol Specification
- [ ] SDK Usage
- [ ] Server Implementation
- [ ] General Implementation
- [ ] Documentation
- [X] Other

### Your Question

Hi, I am honored to be able to use the brilliant Open Protocol called MCP in this new trend these days.

The message below was confirmed in the log.
> Connected to MCP server github!

And the following message is displayed in the CMD window launched together.
> GitHub MCP Server running on stdio

However, after some time, the message below appears on Claude desktop.
> Could not attach to MCP server github
![image](https://github.com/user-attachments/assets/65cb651a-dc3d-46b3-a27b-13ea518f6763)

OS: Windows 10 Pro
Node: v22.11.0
Python: v3.12.6
Location: Republic of Korea
Claude plan: Free

What am I missing? 😥

**Comment by TimHuey** [comment]
Apparently a mac.

**Reply by enchic** [reply]
Ah, now it's really time to buy one, even if it's used. 😏

**Comment by TimHuey** [comment]
Is there a "significance" in the verbiage of the MCP log saying..
2024-12-02T02:13:02.017Z [info] Attempting to connect to MCP server filesystem...
2024-12-02T02:13:02.020Z [info] Attempting to connect to MCP server brave-search...
2024-12-02T02:13:02.063Z [info] Connected to MCP server filesystem!
2024-12-02T02:13:02.069Z [info] Connected to MCP server brave-search!
And the Desktop app saying
![image](https://github.com/user-attachments/assets/04a29a7b-6df1-4b57-a4a6-3705f062a8e3)

I'm wondering if the Desktop app is "connected" to the server, but not "attached" to the server and if that could lead someone to the problem that is occurring and maybe a fix?

**Comment by jspahrsummers** [comment]
Are you running into [modelcontextprotocol/servers#40](https://github.com/modelcontextprotocol/servers/issues/40)? Do any of the workarounds in that thread help?
